text/0068-re-run-sequence.md

```diff
@@ -0,0 +1,25 @@
+# Re-run sequence execution
+
+## Short abstract
+_._
+
+## Why
+### Target audience pain points
+
+As an *SRE/DevOps engineer* using Keptn 
+
+* 
+
+## Investment / Use-case drivers
+
+* 
```

Comment:
```suggestion
Re-triggering is a core functionality of an automation platform to re-run an automated process.
It would help help Keptn newcomers to experience a good automation support.
It would also help to support SRE/DevOps, who are building integrations,
when developing and testing their integration by having a re-run functionality.
```

Reply:
Fixed it.